### PR TITLE
Fix sidebar resizing, when zoomed in on touch device

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 89070026

## Summary

Fixes a small bug when trying to resize the sidebar on a zoomed in device

## Dependencies

NA

## Testing

Use a doccarchive with a sidebar index in it. - 
[SlothCreator.doccarchive.zip](https://github.com/apple/swift-docc-render/files/8154866/SlothCreator.doccarchive.zip)

Steps:
1. Change the theme-settings.json to enable the sidenav
2. Go to `documentation/slothcreator`
3. Zoom in on your mobile device, iPad or using the Simulator
4. Start resizing the sidebar
5. Assert it works properly.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: tests/unit/components/AdjustableSidebarWidth.spec.js [hunk 1]
```diff
@@ -382,4 +382,28 @@ describe('AdjustableSidebarWidth', () => {
...
+    // assert dragging
+    wrapper.find('.resize-handle').trigger('touchstart', { type: 'touchstart' });
+    document.dispatchEvent(createEvent(eventsMap.touch.move, {
+      touches: [{
+        clientX: 300,
+      }],
+    }));
+    // assert class
+    expect(aside.classes()).toContain('dragging');
+    // offset is 100, so we remove it from the clientX, but we add the scrollX.
```
Comment (hqhhuang) [inline tests/unit/components/AdjustableSidebarWidth.spec.js:399]: Could you explain what the offset is referring to here? Is it the `sidebar.offsetLeft`?

**Review by hqhhuang — APPROVED** [review]
Tested the behavior on iPad simulator and desktop, looks good! Could test on mobiles, since the viewport is too narrow to trigger the sidebar resizing.

File: tests/unit/components/AdjustableSidebarWidth.spec.js (hunk 1 above)
Comment (dobromir-hristov) [reply tests/unit/components/AdjustableSidebarWidth.spec.js:399]: the `element.offsetLeft`

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Pull request merged**